dev/start-anvil-and-load-state

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -euo pipefail
+
+export ANVIL_RPC_URL="127.0.0.1:8545"
+
+anvil $@ &
+
+ANVIL_PID=$!
+
+trap "echo 'Caught SIGTERM, forwarding to $ANVIL_PID'; kill -TERM \"$ANVIL_PID\"" SIGTERM
+trap "echo 'Caught SIGINT, forwarding to $ANVIL_PID'; kill -INT \"$ANVIL_PID\"" SIGINT
+
+sleep 2
+
+curl -H "Content-Type: application/json" -X POST --data @anvil-state-load-data.json $ANVIL_RPC_URL
```

Comment:
Do we need this anymore with a compatible state dump? Can't we just start anvil normally and use the --load-state flag?